e2e/test/scenarios/models/models.cy.spec.js

```diff
@@ -520,33 +523,43 @@ describe("scenarios > models", () => {
   });
 
   describe("listing", () => {
+    const modelDetails = {
+      name: "Orders Model",
+      query: {
+        "source-table": ORDERS_ID,
+        limit: 5,
+      },
+      dataset: true,
+    };
```

Comment:
No need to rely on the Orders question with thousands of rows, when we can quickly create a new model that's much lighter in comparison.